<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_4D</class>
 <widget class="QDialog" name="display_4D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>V0 = (0,0,0,0,0)</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="Display_stat_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;F(x,y,z,a) = 9x + 8; 5y; 9+8z; 8/x+a&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Max = 18,9</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
